content/03.methods.md

```diff
@@ -106,10 +106,23 @@ Per recommendations by the New York Genome Center [@doi:10.1101/623702], the Lan
 
 We filtered outputs from both callers on the "PASS" filter, and annotated using The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, and created MAFs using MSKCC's vcf2maf [@url:https://github.com/mskcc/vcf2maf] v1.6.17.
 
+#### Consensus SNV Calling
+
+We considered mutations to be the same across callers if they were identical for the following MAF fields: `Chromosome`, `Start_Position`, `Reference_Allele`,  `Allele`, and `Tumor_Sample_Barcode`.
+We compared all four callers in their overlap with each other, variant allele frequencies of their calls, base changes called, and genomic regions of their calls [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/compare_snv_callers_plots.Rmd].
+VarDict called numerous variants with low allele frequency [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/vaf_violin_plot.png] in addition to nearly all of the calls identified by the other three callers [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/upset_plot.png].
+Because our filtered set was based on the intersection, we removed it from the consensus calculation.
```

Comment:
```suggestion
We termed mutations shared among the other three callers (Strelka2, Mutect2, and Lancet) to be consensus mutation calls and dropped VarDict due to concerns about its call accuracy in this setting.
In practice because our filtered set was based on the intersection of these three sets and because VarDict called nearly every mutation from the other three callers plus many that were unique to it, the decision to not consider VarDict calls has little impact on the results.
```